src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/DynamicFiles/RazorDynamicFileInfoProvider.cs

```diff
@@ -201,7 +201,7 @@ public void SuppressDocument(ProjectKey projectKey, string documentFilePath)
             throw new ArgumentNullException(nameof(documentFilePath));
         }
 
-        if (_lspEditorFeatureDetector.IsLspEditorEnabledAndAvailable(documentFilePath))
+        if (_lspEditorFeatureDetector.IsLspEditorEnabled())
```

Comment:
🐞 This was a regression in the recent fix to _my_ regression. :smile: Here's what it was before all of these changes to `LspEditorFeatureDetector`:

https://github.com/dotnet/razor/blob/d8a8e5788ac1a2779c8b80dab38a5aaf86207d27/src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/DynamicFiles/RazorDynamicFileInfoProvider.cs#L204

Previously, we did not pass the `documentFilePath` here, and doing so has repercussions because we'll start resolving project capabilities here. The repercussions are problematic because this method should always be called on a thread pool thread, and project capability resolution calls into VS COM interfaces that will marshal to the UI thread.